webvtt.html

```diff
@@ -1171,7 +1168,7 @@
     contain the substring "<code>--></code>" (U+002D HYPHEN-MINUS, U+002D HYPHEN-MINUS, U+003E
     GREATER-THAN SIGN).</p>
 
-    <p>A <dfn>WebVTT cue</dfn> consists of the following components, in the given order:</p>
```

Comment:
To elaborate, "Zero or more WebVTT cues and WebVTT comments separated from each other by one or more WebVTT line terminators." would be nonsensical if "WebVTT cue" was a concept and not a piece of syntax.